src/phantom.js

```diff
@@ -77,7 +77,6 @@ export default class Phantom {
             logger.error(`Could not spawn [${phantomjs.path}] executable. Please make sure phantomjs is installed correctly.`);
             logger.error(error);
             this.kill(`Process got an error: ${error}`);
-            process.exit(1);
```

Comment:
@elias-winberg I don't think this is a redundant exit call. `process.exit(1)` exits the node process while `this.process.exit()` exits the PhantomJS process. Since this is confusing, maybe this pull request should fix variable names. Like change `kill` to `killPhantomProcess`? /cc @zovt